ONSdigital/eq-author
# Move sections to own page

### What is the context of this PR?

[Trello card](https://trello.com/c/SAkVHdop/398-adding-section-onto-their-own-page)

### How to review 

1. Open a questionnaire
2. Confirm navigation between pages/sections
3. Confirm deleting sections works as expected
4. Check auto focus works as expected
